andela/pulse-mobile-app
# feat(font): add DIN Pro font support

#### What does this PR do?
Add DIN Pro font support.

#### Description of Task to be completed?
Linking DIN Pro custom font to IOS and Android

#### How should this be manually tested?
- Launch app.
- Use  **fontFamily: 'DINPro-Regular'** to style text.
- Observe that the font is applied

#### What are the relevant pivotal tracker stories?
[#156619134](https://www.pivotaltracker.com/story/show/156619134)

#### Screenshots (if appropriate)
<img width="344" alt="ios_dinpro" src="https://user-images.githubusercontent.com/27197959/39240944-f9e9096c-488d-11e8-9b77-c16a87160edd.png">
<img width="431" alt="android_dinpro" src="https://user-images.githubusercontent.com/27197959/39240953-ffe2e0cc-488d-11e8-85ac-39433d1bb713.png">



#### Check List

#### Questions:
